dev/com.ibm.ws.cdi.ejb_fat/fat/src/com/ibm/ws/cdi12/fat/tests/EjbMiscTest.java

```diff
@@ -27,54 +30,79 @@
 
-       apps[0] = ShrinkWrap.create(WebArchive.class, "multipleWar1.war")
+            WebArchive multipleWarOne = ShrinkWrap.create(WebArchive.class, "multipleWar1.war")
                         .addClass("test.multipleWar1.TestServlet")
                         .addClass("test.multipleWar1.MyBean")
                         .add(new FileAsset(new File("test-applications/multipleWar1.war/resources/WEB-INF/ejb-jar.xml")), "/WEB-INF/ejb-jar.xml")
                         .add(new FileAsset(new File("test-applications/multipleWar1.war/resources/WEB-INF/beans.xml")), "/WEB-INF/beans.xml")
                         .addAsLibrary(multipleWarEmbeddedJar);
 
-       apps[1] = ShrinkWrap.create(WebArchive.class, "multipleWar2.war")
+            WebArchive multipleWarTwo = ShrinkWrap.create(WebArchive.class, "multipleWar2.war")
                         .addClass("test.multipleWar2.TestServlet")
                         .addClass("test.multipleWar2.MyBean")
                         .add(new FileAsset(new File("test-applications/multipleWar2.war/resources/WEB-INF/ejb-jar.xml")), "/WEB-INF/ejb-jar.xml")
                         .add(new FileAsset(new File("test-applications/multipleWar2.war/resources/WEB-INF/beans.xml")), "/WEB-INF/beans.xml")
                         .addAsLibrary(multipleWarEmbeddedJar);
 
-       apps[2] = ShrinkWrap.create(WebArchive.class, "ejbScope.war")
+            WebArchive ejbScope = ShrinkWrap.create(WebArchive.class, "ejbScope.war")
                         .addClass("com.ibm.ws.cdi12.test.ejb.scope.PostConstructingStartupBean")
                         .addClass("com.ibm.ws.cdi12.test.ejb.scope.PostConstructScopeServlet")
                         .addClass("com.ibm.ws.cdi12.test.ejb.scope.RequestScopedBean");
 
-       return apps;
+            server.setMarkToEndOfLog(server.getDefaultLogFile());
+            ShrinkHelper.exportDropinAppToServer(server, multipleWarOne);
+            ShrinkHelper.exportDropinAppToServer(server, multipleWarTwo);
+            ShrinkHelper.exportDropinAppToServer(server, ejbScope);
+
+            assertNotNull("multipleWarOne started or updated message", server.waitForStringInLogUsingMark("CWWKZ000[13]I.*multipleWar1"));
+            assertNotNull("multipleWarTwo started or updated message", server.waitForStringInLogUsingMark("CWWKZ000[13]I.*multipleWar2"));
+            assertNotNull("ejbScope started or updated message", server.waitForStringInLogUsingMark("CWWKZ000[13]I.*ejbScope"));
+            server.setMarkToEndOfLog(server.getDefaultLogFile());
+        } else { 
+            assertNotNull("multipleWarOne started or updated message", server.waitForStringInLogUsingMark("CWWKZ000[13]I.*multipleWar1"));
+            assertNotNull("multipleWarTwo started or updated message", server.waitForStringInLogUsingMark("CWWKZ000[13]I.*multipleWar2"));
+            assertNotNull("ejbScope started or updated message", server.waitForStringInLogUsingMark("CWWKZ000[13]I.*ejbScope"));
+         
```

Comment:
If you wanted to avoid having to track whether you've done the setup before, you can use `ShrinkHelper.exportDropinAppToServer(server, app, DeployOptions.SERVER_ONLY)` which will not put the app in the `published` directory.